src/44bsd/flow_table.cpp

```diff
@@ -402,10 +402,12 @@ CTcpFlow * CFlowTable::alloc_flow(CPerProfileCtx * ctx,
     return(flow);
 }
 
-void CFlowTable::free_flow(CFlowBase * flow){
+void CFlowTable::free_flow(CFlowBase * flow, bool on_table){
     assert(flow);
     flow->m_ctx->m_flow_cnt--;
-    flow->m_ctx->on_flow_close();
+    if (on_table) {
```

Comment:
I think it's a remained part from per profile flow deletion. I'll recover it.

Reply:
> > Now I'd like to get confirmed by the regression test on Travis CI build.
> > What is the keyword triggering the Travis CI?
> > In the wiki page https://github.com/cisco-system-traffic-generator/trex-core/wiki/Travis-CI, I can see `run_this_on_travis` in text and `test_this_on_travis` in the picture.
> 
> > Hi Hanoh,
> > I built with gcc7 --sanitized again and tested as you mentioned.
> > The result is the same with gcc6 --sanitized in my branch.
> > Did you check the error messages you've seen?
> > As I mentioned just before, the error messages are meaningless because CAstfDB instance is not freed at the end.
> > Do you want to remove the error messages from sanitizer?
> 
> CAstfDB i should be freed and you should not see any error with sanitizer
> Could you confirm that you fixed with this branch?

Yes, I made it freed at global destructor.

Reply:
> I think it's a remained part from per profile flow deletion. I'll recover it.

I think my reply is not appropriate for your comment.
Since the source is old, I removed that part already.